LR_Tourist/TouristWebApp/Views/Shared/_Layout.cshtml

```diff
@@ -1,4 +1,7 @@
-﻿<!DOCTYPE html>
+﻿@using Microsoft.AspNetCore.Identity
+@inject SignInManager<IdentityUser> SignInManager
+@inject UserManager<IdentityUser> UserManager
+<!DOCTYPE html>
```

Comment:
Why do we need DOCTYPE here?

Reply:
by default because it goes for all views

Reply:
You don't need this attribute in views at all.